src/test/java/uk/gov/hmcts/reform/professionalapi/controller/request/controller/request/OrganisationCreationRequestValidatorTest.java

```diff
@@ -101,17 +101,17 @@ public void testCallsAllValidators() {
         assertThat(OrganisationCreationRequestValidator.contains("pend")).isEqualTo(false);
     }
 
-    @Test(expected = EmptyResultDataAccessException.class) //null value should throw empty exception
+    @Test(expected = InvalidRequest.class) //null value should throw invalid request exception
     public void validateOrganisationIdentifierNullTest() {
         organisationCreationRequestValidator.validateOrganisationIdentifier(null);
     }
 
-    @Test(expected = EmptyResultDataAccessException.class) //value less than 7 char in length should throw empty exception
+    @Test(expected = InvalidRequest.class) //value less than 7 char in length should throw invalid request
     public void validateOrganisationIdentifierTooShortTest() {
         organisationCreationRequestValidator.validateOrganisationIdentifier("AB");
     }
 
-    @Test(expected = EmptyResultDataAccessException.class) //incorrect format should throw empty exception
+    @Test(expected = InvalidRequest.class) //incorrect format should throw invalid request exception
     public void validateOrganisationIdentifierWrongFormatTest() {
         organisationCreationRequestValidator.validateOrganisationIdentifier("@@@@@@@");
     }
```

Comment:
is n't there a test for length more than 7 characters?

Reply:
I believe there is, I will check, if not I will add it.

Reply:
Addressed in this commit: https://github.com/hmcts/rd-professional-api/pull/442/commits/a46046fb012a380bc963147eba1c7f3fea65fd22